### PR TITLE
322-First-test-should-direct-students-to-read-class-comment

### LINKED ISSUE #322: First test should direct students to read class comment
Only after spending days working on FrameTests for the Bowling exercise did I notice that that BowlingTest has a really great class comment.  

It may be useful for the comment of the first test direct students to read the class comment... "Review the problem specification in the class [Comment] tab" ?

### PR DESCRIPTION
**Pull request by bencoman**
Fixes #322 First test should direct students to read class comment

### CODE REVIEW

File: dev/src/ExercismDev/ExercismExerciseGenerator.class.st [hunk 1]
```diff
@@ -330,8 +330,11 @@ ExercismExerciseGenerator >> generateTestMethodsOn: testClass calling: testVaria
 { #category : #generation }
 ExercismExerciseGenerator >> generateTestNamed: testName in: aClass variable: variableName selector: aSelectorPrefix parameters: testParameters expecting: testResults [ 
 	| output src |
+
 	output := (WriteStream on: '').
-	output nextPutAll: testName; cr; tab; nextPutAll: '| result |'; cr; cr.
+	output << testName; crlf.
+	testCounter = 1 ifTrue: [ output tab << '"Review the class [Comment] tab"' ; crlf].
```
Comment (macta) [inline dev/src/ExercismDev/ExercismExerciseGenerator.class.st]: Maybe - "Tip: Remember to review..."?

**Comment by macta** [comment]
The failing test is my fault - I messed up Die, it was unlocked by HelloWorld but that was temporary and there was a test modified to get it thourhg. @roelof is having a go at a fix

File: dev/src/ExercismDev/ExercismExerciseGenerator.class.st (hunk 1 above)
Comment (bencoman) [reply dev/src/ExercismDev/ExercismExerciseGenerator.class.st]: Cool. Done.

**Comment by macta** [comment]
If you now merge with master, it will fix the broken test (sorry about that)

**Pull request merged**